instagram/instagram/src/Components/Comments/commentapp.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import CommentList from './commentList';
+import CommentForm from './commentForm';
+
+class Commenting extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      comments: [],
+      comment: ''
+    };
+}
+
+addComment = event => {
+  event.preventDefault();
+  const newComment = {task: this.state.comment, completed: false, id: Date.now() };
+  this.setState({
+    comments: [...this.state.comments, newComment],
```

Comment:
I would have the comments updated in the `App.js` I think we got that set up yesterday. if we did we can ignore this comment